run-pylint-check.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
Why do we need this file?

Reply:
To enable running checks done by this command, before creating a PR

Reply:
Ok but I don't think we need it in the repo. Can you please remove it.

Reply:
Done! Removed!